app/src/org/commcare/dalvik/activities/AppManagerActivity.java

```diff
@@ -0,0 +1,169 @@
+package org.commcare.dalvik.activities;
+
+import org.commcare.android.adapters.AppManagerAdapter;
+import org.commcare.android.util.SessionUnavailableException;
+import org.commcare.dalvik.R;
+import org.commcare.dalvik.application.CommCareApplication;
+import org.commcare.dalvik.services.CommCareSessionService;
+
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.AdapterView.OnItemClickListener;
+import android.widget.ListView;
+import android.widget.Toast;
+
+/**
+ * The activity that starts up when a user launches into the app manager.
+ * Displays a list of all installed apps, each of which can be clicked to launch
+ * the SingleAppManagerActivity for that app. Also includes a button for
+ * installing new apps.
+ *
+ * @author amstone326
+ */
+
+public class AppManagerActivity extends Activity implements OnItemClickListener {
```

Comment:
CommCareActivity is a bit of a misnomer, it's not really specific to CommCare apps in any way (Shouldn't rely on a session and such) it's mostly there to remove boilerplate code like linking together ID's and views, and managing state when a dialog rotates and such.

Not a huge deal, but might save work in the future. It is true that here it would really only clear out one or two calls.